docs/pages/web/modalalert.js

```diff
@@ -298,6 +299,61 @@ export default function ModalAlertPage({ generatedDocGen }: {| generatedDocGen:
           />
         </MainSection.Subsection>
       </MainSection>
+
+      <MainSection
+        name="Mobile"
+        description={`ModalAlert requires [DeviceTypeProvider](/web/utilities/devicetypeprovider) to enable its mobile user interface. The example below shows the mobile platform UI and its implementation.
+
+For mobile, all \`size\` are unified into a full mobile viewport Modal. Notice that \`subHeading\` gets moved from the header to the main content container.
```

Comment:
Same nits here. (This is really making me want to build that modular docs thing! Imagine only writing this once 😍)